src/js/embed.js

```diff
@@ -59,51 +60,106 @@ function getQueryParams() {
     return params;
 }
 
+
 window.init = function init(parentEl) {
     const params = getQueryParams();
-    const { sheet, id, format = 'flat' } = params;
-
-    if (!formats[format]) {
-        throw new Error(`format ${format} is not valid`);
-    }
-    const { template, postRender, preprocess, url } = formats[format];
+    const isTailored = Boolean(params.tailored);
```

Comment:
let's go for `!!` in accordance with style guide: https://github.com/guardian/interactive-brexit-companion/blob/master/style-guide.md#coercion--booleans